apps/builder/app/builder/shared/assets/asset-upload.tsx

```diff
@@ -1,13 +1,33 @@
 import { type ChangeEvent, useRef } from "react";
-import { Button, Flex, Tooltip } from "@webstudio-is/design-system";
+import { Button, Flex, Tooltip, toast } from "@webstudio-is/design-system";
 import { UploadIcon } from "@webstudio-is/icons";
-import type { AssetType } from "@webstudio-is/asset-uploader";
+import {
+  type AssetType,
+  MAX_UPLOAD_SIZE,
```

Comment:
Not current PR issue, but MAX_UPLOAD_SIZE should be builder env variable and not a const inside asset-uploader